src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md

```diff
@@ -156,6 +156,17 @@ By default, the Galasa Ecosystem Helm chart creates a Kubernetes Secret containi
 
 For more information on configuring Dex, see the  <a href=https://dexidp.io/docs target="_blank"> Dex documentation</a>.
 
+## Configure the 'owner' of the Galasa service
+When the Galasa service starts up, if a user is configured as one of the "owners" of the Galasa service logs in, they will have administration rights throughout the Galasa service.
```

Comment:
I don't understand this bit of the sentence: `if a user is configured as one of the "owners" of the Galasa service logs in`. Is it meant to be `if a user who is configured as one of the "owners" of the Galasa service logs in`?

Reply:
Re-worded whole section.